README.md

```diff
@@ -1 +1,81 @@
-# th2-conn-ws-client
\ No newline at end of file
+# WebSocket Client
+
+This microservice allows sending and receiving of messages via WebSocket protocol
```

Comment:
This microservice allows sending and receiving **messages** via WebSocket protocol.